homeassistant/components/elmax/switch.py

```diff
@@ -0,0 +1,83 @@
+"""Elmax switch platform."""
+from typing import Any
+
+from elmax_api.model.command import SwitchCommand
+from elmax_api.model.panel import PanelStatus
+
+from homeassistant.components.elmax import ElmaxCoordinator
+from homeassistant.components.elmax.common import ElmaxEntity
+from homeassistant.components.elmax.const import DOMAIN
+from homeassistant.components.switch import SwitchEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import HomeAssistantType
+
+
+class ElmaxSwitch(ElmaxEntity, SwitchEntity):
+    """Implement the Elmax switch entity."""
+
+    @property
+    def is_on(self) -> bool:
+        """Return True if entity is on."""
+        if self.transitory_state is not None:
+            return self.transitory_state
+        return self._device.opened
+
+    async def async_turn_on(self, **kwargs: Any) -> None:
+        """Turn the entity on."""
+        client = self._coordinator.http_client
+        await client.execute_command(
+            endpoint_id=self._device.endpoint_id, command=SwitchCommand.TURN_ON
+        )
+        self.transitory_state = True
+        await self.async_update_ha_state()
```

Comment:
We don't need to call the update state in this case?
Maybe you should call the coordinator for an update instead?